From 4b357b955bc8a119e832a5969c5507137f09c276 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Mon, 11 Oct 2021 18:26:31 +0530
Subject: [PATCH] Remove DisableCameraReceiver feature to avoid icon not found
 issue

- Gallery2 app has a feature to disable the camera launcher
  activity if there is no Camera detected in the first
  CameraProvider call during the boot, but this is not
  applicable for cic cloud remote camera. Hence disable
  this feature in Gallery2 app to avoid camera issues in
  the run time.
- Note: This feature doesn't have dependency on camera
  and gallery use cases.

Tracked_on: OAM-99724
Change-Id: I64756107bfa5658230e7e42846dc0651fba31acf
Signed-off-by: Muhammad Aksar <muhammad.aksar@intel.com>
---
 AndroidManifest.xml | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index a08b2bcae..4ed99706c 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -337,12 +337,6 @@
         </activity>
         <service android:name="com.android.gallery3d.app.BatchService" />
 
-        <receiver android:name="com.android.camera.DisableCameraReceiver">
-            <intent-filter>
-                <action android:name="android.intent.action.BOOT_COMPLETED" />
-            </intent-filter>
-        </receiver>
-
         <service android:name="com.android.camera.MediaSaveService" />
     </application>
 </manifest>
-- 
2.33.0

